data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -282,55 +353,71 @@ def _pollution_factor(self) -> bool:
         # poverty level. Source: Census's American Community Survey]
 
-        pollution_criteria = (
-            (
-                self.df[
-                    field_names.RMP_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.NPL_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.TSDF_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
+        polllution_threshold_columns = [
+            field_names.RMP_LOW_INCOME,
+            field_names.SUPERFUND_LOW_INCOME,
+            field_names.HAZARDOUS_WASTE_LOW_INCOME,
+        ]
+
+        rmp_sites = (
+            self.df[field_names.RMP_FIELD + field_names.PERCENTILE_FIELD_SUFFIX]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        )
+
+        npl_sites = (
+            self.df[field_names.NPL_FIELD + field_names.PERCENTILE_FIELD_SUFFIX]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
-        return pollution_criteria & (
+        tsdf_sites = (
             self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-                + field_names.PERCENTILE_FIELD_SUFFIX
+                field_names.TSDF_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
             ]
-            >= self.LOW_INCOME_THRESHOLD
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
+        # individual series-by-series
+        self.df[field_names.RMP_LOW_INCOME] = (
+            rmp_sites & self.df[FPL_200_SERIES]
+        )
+        self.df[field_names.SUPERFUND_LOW_INCOME] = (
+            npl_sites & self.df[FPL_200_SERIES]
+        )
+        self.df[field_names.HAZARDOUS_WASTE_LOW_INCOME] = (
+            tsdf_sites & self.df[FPL_200_SERIES]
+        )
+
+        _increment_total_eligibility_exceeded(
+            polllution_threshold_columns, self.df
+        )
+
+        return self.df[polllution_threshold_columns].any(axis="columns")
+
     def _water_factor(self) -> bool:
         # In Xth percentile or above for wastewater discharge (Source: EPA Risk-Screening Environmental Indicators (RSEI) Model)
         # AND
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
-        return (
-            self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-                + field_names.PERCENTILE_FIELD_SUFFIX
-            ]
-            >= self.LOW_INCOME_THRESHOLD
-        ) & (
+
+        wastewater_threshold = self.df[FPL_200_SERIES] & (
```

Comment:
this logic is off -- looks like you're checking `& self.df[FPL_200_SERIES]` twice, here in `wastewater_threshold` and in `self.df[field_names.WASTEWATER_LOW_INCOME]`

Reply:
Good catch - Missed the fact that the predicate started with `self.df[FPL_200_SERIES]` when I defined `wastewater_threshold`. 👍